### PR TITLE
Adding Open API Postman Collection

### PR DESCRIPTION
**Pull request by ipbyrne**
This PR is for adding the Open API postman json file that can be used to import all of the end-points into postman as shown in the image below. It also updates the documentation showing how you can do this.

IMPORTANT: The links to the `openapi.json` file in the updated Readme.md will not work until this is merged. 
imported-collection

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,8 +27,18 @@ channel automatically.
  
 ## [Open API Specification](https://w3c-ccg.github.io/traceability-interop/)
 
-The spec contains documentation on use cases as well are required and optional API operations.
+The spec contains documentation on use cases as well are required and optional API operations. You can validate that the openapi spec by running `npm run validate-spec`.
```
Comment (TallTed) [inline README.md:30]:
```suggestion
The spec contains documentation on use cases as well as required and optional API operations. You can validate that they conform to the OpenAPI spec by running `npm run validate-spec`.
```

File: README.md [hunk 2]
```diff
@@ -27,8 +27,18 @@ channel automatically.
...
 
+## Importing the Open API Specification Into Postman
+
+If you are using Postman, you can import all of the available API end-points into an easy to use collection by following the instructions below:
+
+1. While inside a Workspace with Postman, you should see an `import` button at the top of your list of collections. Click this to begin the importing process.
+2. After clicking `import`, a dialog will open providing you with a couple different options for importing the collection. The easiest option is to import using a `Link`. To do this, simply add this URL into the input and click `Import`: `https://w3c-ccg.github.io/traceability-interop/openapi/openapi.json`. If you do not want to use this method, you can also use the `Raw text` option by simply copying the json at `https://w3c-ccg.github.io/traceability-interop/openapi/openapi.json` and pasting it inside the input.
```
Comment (TallTed) [inline README.md:37]:
```suggestion
1. While inside a Postman Workspace, you should see an **import** button at the top of your list of collections. Click this to begin the importing process.
2. After clicking **import**, a dialog will open providing you with a couple of options for importing the collection. The easiest is to import using a **Link**. To do this, simply paste the following URL into the input box, and click **Import**: `https://w3c-ccg.github.io/traceability-interop/openapi/openapi.json`. Alternatively, you can use the **Raw text** option by copying the JSON found at `https://w3c-ccg.github.io/traceability-interop/openapi/openapi.json`, and pasting it into the input box.
```

File: package.json [hunk 3]
```diff
@@ -17,6 +17,8 @@
     "format": "lerna run format --stream",
     "format:docs": "prettier --write docs/**/*.html",
     "test:all": "lerna run test:all --stream",
+    "validate-spec": "npx swagger-cli validate docs/openapi/openapi.yml",
+    "preserve": "npx swagger-cli bundle docs/openapi/openapi.yml -o docs/openapi/openapi.json",
```
Comment (OR13) [inline package.json:21]: please document this command or remove it.

**Review by OR13 — CHANGES_REQUESTED** [review]
see comments

File: docs/openapi/schemas/_index.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
 Pet:
-  $ref: "./DidResolutionResponse.yaml"
+  $ref: "./DidResolutionResponse.yml"
```
Comment (OR13) [inline docs/openapi/schemas/_index.yml:2]: yes, thank you!

File: docs/openapi/schemas/_index.yml [hunk 5]
```diff
@@ -1,4 +1,4 @@
 Pet:
```
Comment (OR13) [inline docs/openapi/schemas/_index.yml:1]: Not sure why we have pet here... probably an error.

File: package.json (hunk 3 above)
Comment (mprorock) [reply package.json:21]: This is great actually - was just working through something similar on my side - please do add some docs on this

**Review by mprorock — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (ipbyrne) [reply package.json:21]: @OR13 @mprorock Readme.md has been updated with some documentation regarding these commands.

File: package.json [hunk 6]
```diff
@@ -17,6 +17,8 @@
     "format": "lerna run format --stream",
     "format:docs": "prettier --write docs/**/*.html",
     "test:all": "lerna run test:all --stream",
+    "validate-spec": "npx swagger-cli validate docs/openapi/openapi.yml",
```
Comment (acarnagey) [inline package.json:20]: shouldn't we add swagger-cli to the devDependencies?

Comment (ipbyrne) [reply package.json:20]: Just added it!

**Review by acarnagey — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Pull request merged**